src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -2511,6 +2511,8 @@ export default class AircraftModel {
         // Crossing into the center
         if (this.isControllable) {
             this.callUp();
+            // for reentry, see #993
```

Comment:
missing empty line between assignment and method call

Reply:
done